modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -278,15 +279,19 @@ def veterans_timezone
     def validate_hearing_type_selection
       return if board_review_hearing_selected? && includes_hearing_type_preference?
 
-      source = '/data/attributes/hearingTypePreference'
-      data = I18n.t('common.exceptions.validation_errors')
-
+      # rubocop:disable Layout/LineLength
       if hearing_type_missing?
-        errors.add source, data.merge(detail: I18n.t('appeals_api.errors.hearing_type_preference_missing'))
+        add_error('If boardReviewOption hearing is selected, hearingTypePreference must also be present', title: 'Missing required field',
```

Comment:
Aha! I _knew_ we'd done `errors.add` where the first argument was the pointer _somewhere_.

Let's keep the whole `appeals_api.errors` language key in mind for later - maybe a tech debt ticket to move all these shared error messages into that namespace.